atc/db/log_conn.go

```diff
@@ -21,21 +21,41 @@ type logConn struct {
 	logger lager.Logger
 }
 
+type logDbTx struct {
+	Tx
+
+	logger lager.Logger
+}
+
+func (t *logDbTx) QueryRow(query string, args ...interface{}) squirrel.RowScanner {
+	t.logger.Debug("tx-query-row", lager.Data{"query": strip(query)})
+	return t.Tx.QueryRow(query, args...)
+}
+
```

Comment:
Should we also wrap the `Tx.Query` func ? There are a few uses of it in the concourse code base.

Reply:
Ditto for `Tx.Exec` & `Tx.QueryRowContext`

Reply:
OK If there are usage then we should probably wrap them too.

Reply:
Updated